cmd/vcert/utils.go

```diff
@@ -165,6 +166,36 @@ func fillCertificateRequest(req *certificate.Request, cf *commandFlags) *certifi
 		}
 		req.CsrOrigin = certificate.LocalGeneratedCSR
 	}
+
+	if cf.validDays != "" {
+		validDays := cf.validDays
+
+		data := strings.Split(validDays, "#")
+		days, _ := strconv.ParseInt(data[0], 10, 64)
+		hours := days * 24
+
+		req.ValidityHours = int(hours)
+
+		issuerHint := ""
+		if len(data) > 1 { //means that issuer hint is set
+
+			option := strings.ToLower(data[1])
+
+			switch option {
```

Comment:
I think we should use constants here for the issuer names and hints. If they are required later on, we don't need to specify them again as strings.

Reply:
Fixed on most recent commit